### PR TITLE
Add @@clear-and-update-index-server-index. / Expansions: all on key "collectiveelastic"

### PR DESCRIPTION
**Pull request by ksuess**
Merge https://github.com/collective/collective.elastic.ingest/pull/30 first!

### CODE REVIEW

File: src/collective/elastic/plone/restapi/expansions.py [hunk 1]
```diff
@@ -74,5 +77,10 @@ def __call__(self, expand=False):
         # blocks_plaintext - only for Volto, not for ClassicUI
         # or Dexterity types w/o blocks behavior
         if IBlocks.providedBy(self.context):
-            result["blocks_plaintext"] = self._getBlocksPlainText(self.context)
+            result["collectiveelastic"].update(
```
Comment (jensens) [inline src/collective/elastic/plone/restapi/expansions.py]: I already thought about it on my last refactoring session, but this breaks existing configurations. As a breaking change we need to bump the version to 3.0.0 in order to go forward.

File: src/collective/elastic/plone/restapi/kitsearch/post.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 from collective.elastic.ingest import OPENSEARCH
 from collective.elastic.ingest.client import get_client
 from plone import api
+from pprint import pformat
```
Comment (jensens) [inline src/collective/elastic/plone/restapi/kitsearch/post.py]: Leftover from a debugging session?

File: src/collective/elastic/plone/restapi/kitsearch/post.py [hunk 3]
```diff
@@ -46,27 +47,14 @@ def search(self, data):
...
-        except TransportError as e:
-            self.request.response.setStatus(503)
-            return dict(error=dict(message=e.message))
-        except NotFoundError as e:
-            self.request.response.setStatus(404)
-            return dict(error=dict(message=e.body["error"]["reason"]))
-        except Exception as e:
-            self.request.response.setStatus(500)
-            return dict(error=dict(message=e.message))
+            search_kwargs["size"] = 10
```
Comment (jensens) [inline src/collective/elastic/plone/restapi/kitsearch/post.py:55]: Removing the error handling here is ... why?

**Review by jensens — CHANGES_REQUESTED** [review]

File: src/collective/elastic/plone/restapi/expansions.py (hunk 1 above)
Comment (ksuess) [reply src/collective/elastic/plone/restapi/expansions.py]: I'm fine with leaving it as it is, as it does affect only the collective.elastic bundle but no other code.

File: src/collective/elastic/plone/restapi/kitsearch/post.py (hunk 2 above)
Comment (ksuess) [reply src/collective/elastic/plone/restapi/kitsearch/post.py]: Removed.

File: src/collective/elastic/plone/restapi/kitsearch/post.py (hunk 3 above)
Comment (ksuess) [reply src/collective/elastic/plone/restapi/kitsearch/post.py:55]: Errors are now catched as is:
grafik

grafik

File: README.rst [hunk 4]
```diff
@@ -116,7 +116,7 @@ The proxy index works out of the box in Volto.
 However, in Volto a direct (and much faster) search is possible by using the ``@kitsearch`` endpoint, bypassing the catalog.
 The endpoint takes a native Open-/ ElasticSearch query and returns the results with Plone permission check.
 
-The Volto add-on `volto-searchkit-block <https://github.com/rohberg/volto-searchkit-block/>`_ (based on `react-searchkit <https://www.npmjs.com/package/react-searchkit>`_) provides a configurable block using this endpoint.
+The Volto add-on `@rohberg/volto-searchkit-block <https://github.com/rohberg/volto-searchkit-block/>`_ (based on `react-searchkit <https://www.npmjs.com/package/react-searchkit>`_) provides a configurable block using this endpoint.
```
Comment (jensens) [inline README.rst]: We prefer to skip naming companies outside of the Contributors paragraph. And even there I usually put names only. One needs to follow the link anyway to get more information and the reference to Rohberg gets resolved. 
I know, in the Volto community companies are keen to have their name somewhere in the namespace, but this here is a backend package.

**Review by jensens — CHANGES_REQUESTED** [review]

**Pull request merged**